README.md

```diff
@@ -9,15 +9,15 @@
 [![install size](https://packagephobia.now.sh/badge?p=type-graphql)](https://packagephobia.now.sh/result?p=type-graphql)
 [![gitter](https://badges.gitter.im/type-graphql.svg)](https://gitter.im/type-graphql?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Create GraphQL schema and resolvers with TypeScript, using classes and decorators!
+Create GraphQL schemata and resolvers with TypeScript, using classes and decorators!
```

Comment:
schemata?

Reply:
Since schema is a Greek word, its plural would be schemata but for readability's sake we could use schema as plural just fine.